scripts/create_project_table.py

```diff
@@ -0,0 +1,131 @@
+"""
+Process README files to extract metadata, generate a table of content, 
+and update a central README file with aggregated data.
+
+- Extract metadata from the top section of the README file for each project (e.g., between "---" delimiters).
+- Generate table rows summarizing the metadata
+- Insert the generated rows into the project table in the main README file
+
+Classes:
+- ReadmeMetadataProcessor: Extracts and processes metadata from individual README files.
+"""
+
+import os
+import yaml
+from pathlib import Path
+from typing import List
```

Comment:
No need to import `typing.List` - builtin `list` works too.

```suggestion
import os
from pathlib import Path
import yaml
```